benchmarks/user/advisor/README.md

```diff
@@ -2,35 +2,50 @@ Example runs:
 
 * `python3 run_advisor.py --name isotropic --path <path-to-devito>/examples/seismic/acoustic/acoustic_example.py`
 * `python3 run_advisor.py --name tti_so8 --path <path-to-devito>/examples/seismic/tti/tti_example.py --exec-args "-so 8"`
-* `python3 run_advisor.py --name iso_ac_so6 --path <path-to-devito>/benchmarks/user/benchmark.py --exec-args "bench -P acoustic -so 6 --tn 200 -d 100 100 100 --autotune off -x 1"`
+* `python3 run_advisor.py --name iso_ac_so4 --path <path-to-devito>/benchmarks/user/benchmark.py --exec-args "run -P acoustic -so 4 --tn 200 -d 100 100 100"`
 
-After the run has finished you should be able to plot a roofline with the results and export roofline data to JSON using:
+After the run has finished you should be able to save a .json and plot the
+roofline with the results:
 * `python3 roofline.py --name Roofline --project <advisor-project-name>`
 
 To create a read-only snapshot for use with Intel Advisor GUI, use:
 * `advixe-cl --snapshot --project-dir=<advisor-project-name> pack -- /<new-snapshot-name>`
 
 Prerequisites:
-* Support guaranteed only for Intel Advisor as installed with Intel Parallel Studio v 2020 Update 2
-  and Intel oneAPI 2021; earlier years may not work; other 2020/2021 versions, as well as later years,
-  may or may not work.
+* Support is guaranteed only for Intel oneAPI 2025; earlier versions may not work.
+You may download Intel oneAPI [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?packages=oneapi-toolkit&oneapi-toolkit-os=linux&oneapi-lin=apt).
+
+* Add Advisor (advixe-cl) and compilers (icx) in the path. It should be along the lines of:
+```sh
+source /opt/intel/oneapi/advisor/latest/env/vars.sh
+source /opt/intel/oneapi/compiler/latest/env/vars.sh
+```
+depending on where you installed oneAPI
+
 * In Linux systems you may need to enable system-wide profiling by setting:
-  - `/proc/sys/kernel/yama/ptrace_scope` to `0`
-  - `/proc/sys/kernel/perf_event_paranoid` to `1`
 
-* `numactl` must be available on the system. If not available, install with:
-	`sudo apt-get install numactl`
+```sh
+/proc/sys/kernel/yama/ptrace_scope to 0
```

Comment:
Should these be the actual commands?

Reply:
wdym? They may or may not be always needed

Reply:
As root
```bash
echo 0 > /proc/sys/kernel/yama/ptrace_scope
echo 1 > /proc/sys/kernel/perf_event_paranoid
```
As far as I know this is the only way to achieve this. Downside is you need to be root :confused:

Reply:
Correct @JDBetteridge.
So we have these command there to the discretization of the user.
They may not be needed. This needs for sure sudo rights. We leave it for the user IFF needed
e.g. for me, it works fine with
```
cat /proc/sys/kernel/yama/ptrace_scope
1
cat /proc/sys/kernel/perf_event_paranoid 
4
```

IIIRC it is more needed for vtune rather than advisor